src/qmapshack/dem/IDem.cpp

```diff
@@ -334,6 +349,41 @@ void IDem::slopecolor(QVector<qint16>& data, qreal w, qreal h, QImage &img)
     }
 }
 
+void IDem::elevationlimit(QVector<qint16>& data, qreal w, qreal h, QImage &img, qreal elevationConversion)
+{
+    int wp2 = w + 2;
+
+    for(unsigned int m = 1; m <= h; m++)
+    {
+        unsigned char* scan = img.scanLine(m - 1);
+        for(unsigned int n = 1; n <= w; n++)
+        {
+            qint16 win[eWinsize3x3];
+            fillWindow(data, n, m, wp2, win);
+
+	    // get maximum of window (_not_ mean)
+	    //
+            qreal elevation = -2.0;
+	    for(unsigned int i = 0; i < eWinsize3x3; i++)
```

Comment:
Mind your formatting! You can use uncrustify to cleanup spacing and formatting. Use `call_Uncrustify.cfg` in the project root as configuration.

Reply:
sorry for inserting tabs.
My vim default settings for c++ ;-)
I changed it to spaces